mail/1.6/mail-spec-1.6.html

```diff
@@ -0,0 +1,664 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="generator" content="Asciidoctor 1.5.8">
+<meta name="author" content="Jakarta Mail Team, https://projects.eclipse.org/projects/ee4j.mail">
```

Comment:
Project link seems to actually be at: https://projects.eclipse.org/projects/ee4j.javamail

Reply:
I mean, though the https://github.com/eclipse-ee4j/mail-spec references https://projects.eclipse.org/projects/ee4j.mail that link doesn't work, so maybe you need to contact the Eclipse webmaster if this is the intended URL.

Reply:
I've been waiting for a month for the Eclipse team to rename the project:
https://bugs.eclipse.org/bugs/show_bug.cgi?id=548921
The URLs anticipate the renaming.